fuel/app/themes/default/partials/my_widgets.php

```diff
@@ -83,12 +83,11 @@
 									<span class="undo">Removed <a href="#">Undo</a></span>
 
 									<select ng-disabled="selected.shareable==false" tabindex="0" id="perm" class="perm" ng-model="collaborator.access" ng-change="checkForWarning(collaborator)">
-										<option value={{ACCESS.FULL}} ng-selected="collaborator.access == ACCESS.FULL" >Full</option>
-										<option value={{ACCESS.VISIBLE}} ng-selected="collaborator.access == ACCESS.VISIBLE" >View Scores</option>
+										<option ng-repeat="accessLevel in accessLevels" value={{accessLevel.value}} ng-selected="collaborator.access == accessLevel.value">{{accessLevel.text}}</option>
 									</select>
 
 									<a ng-if="selected.shareable" tabindex="0" class="remove-expiration" role="button" ng-click="removeExpires(collaborator)" ng-show="collaborator.expires">X</a>
-									<span class="expires">Expires: </span><input ng-disabled="!selected.shareable" type="text" class="exp-date user{{::collaborator.id}}" ng-model="collaborator.expiresText" readonly="true" />
+									<span class="expires">Expires: </span><input ng-disabled="!selected.shareable" type="text" class="exp-date user{{::collaborator.id}}" ng-model="collaborator.expiresText" readonly="true" /><p>{{n}}</p>
```

Comment:
What's the purpose behind the `<p>{{n}}</p>`? My guess is it's a debug element, but want to be 100% sure.

Reply:
Seems likely, since there's no `$scope.n` in the Angular half of this.

Just get rid of the debug stuff and this should be ready to go.

Reply:
Oh yeah, it was just for debugging. I must have overlooked it. Sorry.